### DISCUSSION TITLE
Optimization/reduce app size using coreui-vue

### DISCUSSION THREAD

**Discussion by raheelhirani35** [coreui/coreui-vue#245] [Q&A]
How we can reduce app size using coreui and coreui-vue if we are only few components of coreui-vue as app size is exceeding 1.5mb and most part of js is coreui which we are not using. So how to use it with better optimization so it reduce app size.

**Comment by gfoidl** [comment]
Did you import individual components or did you use it "plugin-like"?
The former is smaller in size (by design).
